[2022-05-23 20:16:16,740] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-23T00:00:00+00:00 [queued]>
[2022-05-23 20:16:16,766] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-23T00:00:00+00:00 [queued]>
[2022-05-23 20:16:16,766] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:16:16,766] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-05-23 20:16:16,766] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:16:16,796] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): second_task> on 2022-03-23 00:00:00+00:00
[2022-05-23 20:16:16,808] {standard_task_runner.py:52} INFO - Started process 2925 to run task
[2022-05-23 20:16:16,820] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-23T00:00:00+00:00', '--job-id', '720', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpqdqjhu_z', '--error-file', '/tmp/tmpj6_hbi7t']
[2022-05-23 20:16:16,829] {standard_task_runner.py:80} INFO - Job 720: Subtask second_task
[2022-05-23 20:16:17,020] {task_command.py:369} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-23T00:00:00+00:00 [running]> on host 061a475e6c14
[2022-05-23 20:16:17,224] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=gerson
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-23T00:00:00+00:00
[2022-05-23 20:16:17,225] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-23 20:16:17,225] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-05-23 20:16:17,260] {subprocess.py:85} INFO - Output:
[2022-05-23 20:16:17,263] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-05-23 20:16:17,264] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-23 20:16:17,435] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220323T000000, start_date=20220523T201616, end_date=20220523T201617
[2022-05-23 20:16:17,519] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-23 20:16:17,635] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
